index.js

```diff
@@ -71,4 +71,21 @@ String.prototype.pysplit = function (separator = " ", maxsplit = -1) {
     : tokens.slice(0, maxsplit).concat(concatenation);
 };
 
+String.prototype.center = function (width, fillchar = ' ') {
+  if (typeof width !== 'number') {
+    throw new TypeError("'width' must be a number");
+  }
+  if (typeof fillchar !== 'string' || fillchar.length !== 1) {
+    throw new TypeError("The fill character must be exactly one character long");
+  }
+  if (this.length >= width) {
+    return this.toString();
+  }
+  const padding = width - this.length;
+  const leftPadding = Math.floor(padding / 2);
+  const rightPadding = padding - leftPadding;
+  const centeredText = fillchar.repeat(leftPadding) + this + fillchar.repeat(rightPadding);
+  return centeredText;
```

Comment:
I noticed this weird behaviour:
<img width="953" alt="image" src="https://github.com/Aritra8438/jsstring/assets/64671908/4829c1d1-40f8-4073-a2c9-df5f4196c8bd">

Can you look into it?